src/main.c

```diff
@@ -273,10 +275,19 @@ bool save_stream(char user_board[], char plain_board[], int n)
 	return true;
 }
 
+void generate_stream_output(int difficulty) {
+	char* stream;
```

Comment:
please use one line for these two lines.

Reply:
You probably mean the two lines about the stream variable, and not the function header and the variable declaration, so i did that.